flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/ResultPartitionManager.java

```diff
@@ -79,8 +128,47 @@ public ResultSubpartitionView createSubpartitionView(
         return subpartitionView;
     }
 
+    @Override
+    public ResultSubpartitionView createSubpartitionViewOrNotify(
```

Comment:
rename to something like `registerPartitionRequestListener`,`registerPartitionRequestNofitifer`?